### ISSUE TITLE
Recognize holder name with number inside

### ISSUE THREAD

**Issue by elbelga** [faceterteam/PayCards_iOS_Source#6]
Is it possible to recognize holder name with numbers inside?

**Comment by rostikjooo** [comment]
yes, in PayCards_Source-master/PayCardsRecognizer/CrossPlatform/Recognizer/NameRecognizer.cpp add number characters in "alphabet" variable
